Timeline/package.json

```diff
@@ -16,7 +16,7 @@
     "dist": "npm run check && webpack --config-name=distProd",
     "dist:dev": "npm run check && webpack --config-name=distDev",
     "dist:watch": "npm run check && webpack --config-name=distDevWatch",
-    "check": "check-peer-dependencies --yarn",
+    "check": " ",
```

Comment:
why remove this?

Reply:
I had it by passed in my machine. This change got leaked. I will add this back.